packages/components/slideshow/src/less/slideshow.less

```diff
@@ -91,8 +91,9 @@
     z-index: 2;
 
     .oui-modal__close-button {
-      top: 0;
-      right: 0;
+      color: @oui-slideshow-title-font-color;
+      top: 5px;
```

Comment:
Couldn't be stored as a variable into following file?
- `packages/base/core/src/less/variables/components/slideshow.less`

`px` unit is mainly used to border size. Could you please use `rem-calc`?